### ISSUE TITLE
Arachnoid granulation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8554]
**Preferred term label:**
Arachnoid granulation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Arachnoid granulations (AGs) are tufts of arachnoid membrane invaginated into the dural sinuses through which cerebrospinal fluid (CSF) enters the venous system. The lesions are primarily located in the parasagittal region along the superior sagittal sinus, which is occasionally seen at the transverse sinus. Normally dural venous sinus AGs typically range from 2 to 8 mm in size, but may grow enough to expand the dural sinuses, and even the inner table, diploic space and outer table of the skull.
PMID:22900137

**Parent term (use hpo.jax.org/app)**
Abnormal arachnoid mater morphology HP:0100700

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
618504 
PMID: 31079898  1/3

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
